### ISSUE TITLE
Fofdata needs some extra parameters

### ISSUE THREAD

**Issue by Samwalton9** [jmackereth/eaglestuff#15]
Just a note that R200 would be useful to have in the FoF data output for scaling.

We could probably do with some other parameters carried through, but that's one that I just went to look for.

**Comment by jmackereth** [comment]
isn't this already saved in the hd5 files? it is in the version i have anyway, ill check the latest one on here.

**Comment by jmackereth** [comment]
yeah, the savehaloarrays function does save the r200. there is probably more things we should carry through though.

I have added a sersic profile fit to the latest version which returns the scale length and the sersic index of the halo too.

**Comment by Samwalton9** [comment]
Ah, just checked and this is in the most recent version of the code and not the one I have. Will chat about versions tomorrow, need to check if I made any changes to the version I have.
